aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>Audio --&gt; audio</p><p>it's --&gt; its</p><p> </p><p>Define FFT. It's never mentioned in the notebook until now.</p><p>You might want to link to something about cepstrum if you don't want to go into it here and you're assuming the reader knows what it is. Also, using the main term you are describing in the description isn't that helpful. "Cepstral features: features of cepstral" :)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>